File: CaesarCipher.py
ALPHABET = ' ABCDEFGHIJKLMNOPQRSTUVWYZ'
KEY = 3


def caesar_encrypt(plain_text):

    cipher_text = ''
    plain_text = plain_text.upper()

    for c in plain_text:
        c='E'

        index = ALPHABET.find(c)


        index = (index + KEY) % len(ALPHABET)

        cipher_text = cipher_text + ALPHABET[index]


        return cipher_text 

File: Caesar_Cipher/crypto_algorithms/CaesarCipher.java
package com.globalsoftwaresupport;

public class CaesarCipher {

    private String ALPHABET = " ABCDEFGHIJKLMNOPQRSTUVWYZ"

    public String encrypt(String plainText, int key) {
String cipherText = "";

plainText = plainText.toUpperCase();

for (int i=0; i<plainText.length(); i++) {
    char character = plainText.charAt(i);

    int charIndex = ALPHABET.indexOf(character);
    int encryptedIndex = (charIndex+key) % ALPHABET.length();

    cipherText += ALPHABET.charAt(encryptedIndex)
}
    return cipherText;
    }
}